orca-mine/src/main/groovy/com/netflix/spinnaker/orca/mine/pipeline/DeployCanaryStage.groovy

```diff
@@ -60,6 +60,11 @@ class DeployCanaryStage extends ParallelDeployStage implements CloudProviderAwar
   }
 
   @Override
+  Class<com.netflix.spinnaker.orca.Task> completeParallelTask() {
+    return CompleteDeployCanaryTask.class;
```

Comment:
The old `ParallelStage` has a `public Task completeParallel()` method in it's interface.